lib/core-integration/src/Test/Integration/Framework/TestData.hs

```diff
@@ -266,9 +267,14 @@ errMsg403TxTooBig n = "I had to select " ++ show n ++ " inputs to construct the\
 errMsg404MalformedTxPayload :: String
 errMsg404MalformedTxPayload = "I couldn't decode the payload that seems to be an\
     \ externally-signed transaction due to: wrongly constructed binary blob. \
-    \Make sure to send a base64-encoded binary blob, in the proper binary \
+    \Make sure to send a hex-encoded binary blob, in the proper binary \
```

Comment:
After seeing `proceed with the payload` below, I thought of a different direction here:

*I couldn't verify that the payload has the correct binary format. I won't send it to the node. Make sure the blob has the correct binary format (and is encoded as hex)*

Reply:
sounds good. corrected to that